flybywiresim/a32nx
# fix(efb): Fixing Keyboard Not Showing On Fuel Page

Fixes #7915 

## Summary of Changes

Removing CSS class allowing for the On-Screen Keyboard to be visible on the Fuel page on the EFB.

## Screenshots (if necessary)


![image](https://user-images.githubusercontent.com/32962022/235262669-433a3bb4-0e9d-4fcd-bb62-27e480a28556.png)

## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

- Load into airplane on the runway
- Go to EFB
- Open Ground section
- Open the Fuel tab
- Click fuel number
- Confirm keyboard is visible


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
